src/utils/storage.ts

```diff
@@ -1,8 +1,31 @@
 import { ApiPromise } from "@polkadot/api";
 import { Codec } from "@polkadot/types/types";
+import { xxhashAsHex, blake2AsHex } from "@polkadot/util-crypto";
 import { AddressOrPair } from "@polkadot/api/types";
 
 import { DefaultTransactionAPI } from "../parachain";
+import { stripHexPrefix } from "./encoding";
+
+export function getStorageKey(moduleName: string, storageItemName: string): string {
+    return xxhashAsHex(moduleName, 128) + stripHexPrefix(xxhashAsHex(storageItemName, 128));
+}
+
+export function blake2_128Concat(data: `0x${string}`): string {
+    return blake2AsHex(data, 128) + stripHexPrefix(data);
+}
+
+export function getStorageMapItemKey(
+    moduleName: string,
+    storageItemName: string,
+    mapItemKey: `0x${string}`,
+    nestedMapItemKey?: `0x${string}`
+): string {
+    let storageKey = getStorageKey(moduleName, storageItemName) + stripHexPrefix(blake2_128Concat(mapItemKey));
+    if (nestedMapItemKey) {
+        storageKey += stripHexPrefix(blake2_128Concat(nestedMapItemKey));
+    }
+    return storageKey;
+}
```

Comment:
Remove, use polkadot-js for this instead. See: https://github.com/interlay/interbtc-api/pull/588

Reply:
`api.query.btcRelay.stableBitcoinConfirmations.key()` <- oh that's very nice!